planning/behavior_path_planner/src/scene_module/scene_module_bt_node_interface.cpp

```diff
@@ -68,7 +68,7 @@ BT::NodeStatus SceneModuleBTNodeInterface::tick()
       // std::exit(EXIT_FAILURE);  // TODO(Horibe) do appropriate handing
     }
     scene_module_->unlockRTCCommand();
-    return BT::NodeStatus::SUCCESS;
```

Comment:
@rej55 
what is this change for?

Reply:
@kosuke55 
If we apply https://github.com/autowarefoundation/autoware.universe/pull/2442, the feature to output multiple execute requests is required.
To run fall back in the behavior tree, I changed the node status.